gen/definitions/policyassignments.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: Policy Assignments
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/assignment/policyassignments
+no_delete: true
+no_data_source: true
+data_source_name_query: true
+doc_category: Policy Assignments
+attributes:
+  - model_name: type
+    type: String
+    value: "PolicyAssignment"
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
+    mandatory: false
+    example: AccessPolicyName
+    data_path: ["policy"]
+    exclude_example: true
+    exclude_test: true    
+  - model_name: type
+    type: String
+    mandatory: false
+    example: AccessPolicy
+    data_path: ["policy"]
+    exclude_example: true
+    exclude_test: true    
+  - model_name: id
+    tf_name: policy_id
+    type: String
+    id: true
+    mandatory: true
+    example: 0050568A-2561-0ed3-0000-004294972836
+    data_path: ["policy"]
+    test_value: fmc_access_control_policy.example.id
+  - model_name: targets
+    type: Set
+    mandatory: true
+    attributes:
+      - model_name: id
+        type: String
+        id: true
+        mandatory: true
+        example: "9862719c-8d5f-11ef-99a6-aef0794da1c1"
+      - model_name: type
+        type: String
+        enum_values: [Device,DeviceHAPair,DeviceGroup]
+        mandatory: true
```

Comment:
Can you confirm this is required?

Reply:
Required:

Error: 406
Response body
Download

{
  "error": {
    "category": "FRAMEWORK",
    "messages": [
      {
        "description": "Entered device type:'null' is either incorrect or not supported in policy assignment."
      }
    ],
    "severity": "ERROR"
  }
}